src/couch_task_status.erl

```diff
@@ -122,6 +127,14 @@ handle_call({add_task, TaskProps}, {From, _}, Server) ->
     [_] ->
         {reply, {add_task_error, already_registered}, Server}
     end;
+handle_call({filter, Key, Value}, _, Server) ->
+    All = [
```

Comment:
Use following to avoid doing the same thing in a tight loop
```
handle_call({filter, Key0, Value0}, _, Server) ->
Key = couch_util:to_list(Key0),
Value = couch_util:to_list(Value0),
```